Integrations/integration-ArcherRSA.yml

```diff
@@ -2920,14 +2920,17 @@ script:
         }
         var offset = tz ? (60 * 1000 * new Date().getTimezoneOffset()) : 0;
         offset = tz ? parseInt(tz) : 0;
```

Comment:
Don't think we need those two lines

Reply:
I agree, thought it would be better to remove them in a rewrite